### ISSUE TITLE
Support leases for File Shares

### ISSUE THREAD

**Issue by MRayermannMSFT** [microsoft/AzureStorageExplorer#3791]
File Shares let you lease files. We should support acquiring and breaking such leases:

https://docs.microsoft.com/en-us/rest/api/storageservices/lease-file

**Comment by MRayermannMSFT** [comment]
I just investigated this, not as simple as though. Will update original issue with details.

**Comment by ishepherd** [comment]
Yes please. I just was hunting for this feature and surprised to find it only in the portal version, I had figured the desktop would be feature-complete if anything was.
